package.json

```diff
@@ -40,6 +40,7 @@
     "babel-preset-react": "~6.11.1",
     "browser-sync": "~2.15.0",
     "browser-sync-webpack-plugin": "~1.1.2",
+    "bs-fullscreen-message": "~1.1.0",
```

Comment:
We use https://github.com/glenjamin/webpack-hot-middleware, it provides a fullscreen overlay showing the error, nice